test-app/app/templates/application.hbs

```diff
@@ -1,4 +1,7 @@
 <h1>Ember Engines Demo</h1>
+<span class='global-refresh-counter'>
```

Comment:
I guess it because the project doesn't have newer prettier config that excludes hds files from adding single quotes and my editor to auto fix on save, I will fix this one np

Reply:
Fix single quotes and added newer config to prettier